201-web-app-github-deploy/azuredeploy.json

```diff
@@ -51,7 +51,7 @@
         },
         "repoURL": {
             "type": "string",
-            "defaultValue": "https://github.com/Azure/azure-quickstart-templates.git",
+            "defaultValue": "https://github.com/Microsoft/project-html-website",
```

Comment:
the readme for this repo says the repo is no longer used?  Do we have an Azure4Students account or repo we could link to?

Reply:
As @leestott  recommended, I'm using this repo: https://github.com/Microsoft/project-html-website

Reply:
@leestott can we find a supported repo for this sample?